### PR TITLE
LMS - Upgrading the AWS provider version to the latest

### PR DESCRIPTION
**Pull request by SKIMRAN0509**
Updated modules repo of IAM-Aggregate-policy and Terraform-Null-label in main.tf
updated lambda version from python3.9 to python3.12

### CODE REVIEW

**Comment by k-agrawal** [comment]
@SKIMRAN0509 This needs to go on `vendored` branch

File: main.tf [hunk 1]
```diff
@@ -79,7 +71,7 @@ data "aws_kms_key" "lambda" {
 module "default_label" {
   enabled = var.enabled
 
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.16.0"
+  source     = "git::https://github.com/teamclairvoyant/terraform-null-label.git?ref=fix/lms-24380-Null-label"
```
Comment (k-agrawal) [inline main.tf]: Use simple cloudposse/label module

File: main.tf [hunk 2]
```diff
@@ -332,7 +325,7 @@ data "aws_iam_policy_document" "user_password_kms_permissions" {
 }
 
 module "aggregated_policy" {
-  source = "git::https://github.com/cloudposse/terraform-aws-iam-policy-document-aggregator.git?ref=tags/0.2.0"
```
Comment (k-agrawal) [inline main.tf:335]: Try using any of the terraform's in-built function

File: main.tf (hunk 1 above)
Comment (SKIMRAN0509) [reply main.tf]: Updated with "cloudposse/label/null"

**Pull request closed without merging**